lib/repository.py

```diff
@@ -81,7 +82,39 @@ def flush(self):
     if graphical:
         splash.Hide()
         sys.stdout = sys.__stdout__
-    print "The retriever is up-to-date"
+    print "\nThe retriever is up-to-date"
+
+
+def update_progressbar(progress):
+    """Shows progressbar given progress in float
+    Example: 0.5 for 50%
+    """
```

Comment:
In the doc string we can remove this `Example:` 0.5 for `50%`
or we wan instead explain how this comes up
 
 """Shows progressbar given progress in float

accepts a a number  between 0 and 1 and it's converted to a float.
a number under 0 represents a 'halt'.
a number at 1 or greater represents 100%
"""